client/components/domains/register-domain-step/index.jsx

```diff
@@ -55,20 +57,25 @@ function processSearchQueue() {
 					continue outerLoop;
 				}
 			}
-			reportSearch( queue[ i ] );
+			reportSearchStats( queue[ i ] );
 		}
 }
 
-function reportSearch( { query, section } ) {
-	analytics.recordEvent( 'searchFormSubmit', query, section );
+function reportSearchStats( { query, section, timestamp } ) {
+	let timeDiffFromLastSearchInSeconds = 0;
+	if ( lastSearchTimestamp ) {
+		timeDiffFromLastSearchInSeconds = Math.floor( ( timestamp.valueOf() - lastSearchTimestamp.valueOf() ) / 1000 );
```

Comment:
I think you can drop the calls to `valueOf` here - especially since the line is crowded as it is :wink: